libraries/AP_Math/location.cpp

```diff
@@ -23,14 +23,16 @@
 #include "AP_Math.h"
 #include "location.h"
 
-// return bearing in centi-degrees between two positions
+// return bearing_rad in radians between two positions
+float get_bearing_rad(const Vector2f &origin, const Vector2f &destination)
+{
+    return wrap_2PI(atan2f(destination.y-origin.y, destination.x-origin.x));
```

Comment:
I think this should be the same 
```suggestion
    return wrap_2PI((destination - origin).angle());
```